### PR TITLE
fixes grilles not updating cables

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

Fixes cables not updating nodes when grilles are destroyed

## Changelog
:cl:
fix: Cable nodes now visually update after grilles are built and destroyed
/:cl:

### CODE REVIEW

**Comment by Akrilla** [comment]
Just since you mentioned grilles, is the whole electrified grille thing when on top of a wire (not a knot) intended?

**Comment by actioninja** [comment]
All wires are knots, yes.

**Comment by actioninja** [comment]
Wait a second I literally made a proc for this already am I fucking stupid

File: code/game/objects/structures/grille.dm [hunk 1]
```diff
@@ -17,6 +17,10 @@
 	var/broken_type = /obj/structure/grille/broken
 	rad_flags = RAD_PROTECT_CONTENTS | RAD_NO_CONTAMINATE
 
+/obj/structure/grille/Destroy()
+	. = ..()
+	update_cable_icons_on_turf(get_turf(src))
```
Comment (AnturK) [inline code/game/objects/structures/grille.dm]: You want this before ..() since it will be nullspace otherwise.

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/grille.dm (hunk 1 above)
Comment (actioninja) [reply code/game/objects/structures/grille.dm]: right, of course

**Pull request merged**